docs/features/software-catalog/index.md

```diff
@@ -40,9 +40,10 @@ browse the catalog at `http://localhost:3000`.
 
 ## Adding components to the catalog
 
-The source of truth for the components in your software catalog are
-[metadata YAML files](descriptor-format.md) stored in source control (GitHub,
-GitHub Enterprise, GitLab, ...).
+The source of truth for the components in your software catalog are [metadata
+YAML files](descriptor-format.md) stored in source control (GitHub, GitHub
+Enterprise, GitLab, ...). Repositories can include multiple metadata files, and
+there's no requirement for metadata files to be placed in the repository root.
```

Comment:
I think that's great context & I'm wondering if we would need to say one metadata file in the repo root is still the most common case as it will be mostly covered like the rest of the documentation. E.g. smth like this:

```suggestion
Enterprise, GitLab, ...). Repositories can include one or multiple metadata files. Usually the metadata file is located in the repository root. This is not a formal requirement & metadata files can be placed anywhere in the repository.
```

Reply:
Good suggestion! done